test/rest-coercion/_jsonbody.context.js

```diff
@@ -0,0 +1,73 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
```

Comment:
I have two sets of files: helper files that does not contain any tests (names starts with underscode), and ".suite.js" files with the tests. The code loading all tests files is skipping underscore files.

I don't mind using a different approach/convention, can you propose a better solution?